pkg/client/appliance.go

```diff
@@ -8,15 +8,7 @@ import (
 )
 
 func (h *V1Client) GetApplianceByName(deviceName string) (*models.V1EdgeHostDevice, error) {
```

Comment:
Consider removing this method altogether if lookup by name is not supported.

Reply:
@TylerGillson could you please elaborate on your comment? GetApplianceByName() is used for a single appliance lookup by name and for appliance**s** lookup we will use tags/labels without a name.

Reply:
Nvm - my bad. I didn't see any usages of `deviceName` in the diff and thought that that var was unused.